### PR TITLE
Added activation dynamic from De Groote 

### PR DESCRIPTION
**Pull request by leasanchez**

### CODE REVIEW

File: src/Muscles/StateDynamicsDeGroote.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        bool alreadyNormalized){
+    // From DeGroote https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5043004/
+
+    // Get activation and excitation and make difference
+    biorbd::utils::Scalar diff; //(e - a)
+    biorbd::utils::Scalar f;    //activation dynamics
+    //std::tanh;
+
+    diff = *m_excitation - *m_activation;
+    f = 0.5 * tanh(0.1*diff);
```
Comment (pariterre) [inline src/Muscles/StateDynamicsDeGroote.cpp:47]: Could you confirm that "tanh" is effectively tan?

File: src/Muscles/StateDynamicsDeGroote.cpp [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    
+    biorbd::utils::Scalar denom_activation;   // dénominateur for activation
+    biorbd::utils::Scalar denom_deactivation; // dénominateur for deactivation
+
+    denom_activation = characteristics.torqueActivation()   * (0.5+1.5* *m_activation);
+    denom_deactivation = characteristics.torqueDeactivation() / (0.5+1.5* *m_activation);
```
Comment (pariterre) [inline src/Muscles/StateDynamicsDeGroote.cpp:53]: No need to separate the declaration and affectation here

Comment (leasanchez) [reply src/Muscles/StateDynamicsDeGroote.cpp]: done

File: src/Muscles/StateDynamicsDeGroote.cpp (hunk 1 above)
Comment (leasanchez) [reply src/Muscles/StateDynamicsDeGroote.cpp:47]: it should be the hyperbolic tangent function, it is the function used in the article

**Pull request merged**